api/internal/store/entity/live.go

```diff
@@ -31,6 +31,7 @@ type Live struct {
 	Canceled     bool           `gorm:""`                     // 配信中止フラグ
 	StartAt      time.Time      `gorm:""`                     // 配信開始日時
 	EndAt        time.Time      `gorm:""`                     // 配信終了日時
+	ChannelArn   string         `gorm:""`                     // チャンネルArn
```

Comment:
クエリ見てると、gormタグにはデフォルトNULL指定つけておく方が良さそうかも..?

```suggestion
	ChannelArn   string         `gorm:"default:null"`                     // チャンネルArn
```